docs/t-sql/functions/char-transact-sql.md

```diff
@@ -176,8 +176,8 @@ single_byte_representing_complete_character single_byte_representing_complete_ch
 ｼ                                           ｼ                                         
 ```
 
-### F. Using CHAR to return multibyte characters  
-This example uses the an integer and hex values in the valid range for ASCII. However, the CHAR function returns NULL because the parameter represents only the first byte of a multibyte character.
+### F. CHAR inability to return multibyte characters
+This example uses integer and hex values in the valid range for ASCII. However, the CHAR function returns NULL because the parameter matches the first byte of a number of multibyte characters in the collation, but no single complete character.

```

Comment:
Keep original form.

Reply:
I reverted to the original title and sentences, but I've added a line to address character composition. I assume that that's the user intent that drives this example.

Reply:
This still bothers me. Wouldn't it make sense to draw a distinction here between the abstraction level here? If `CHAR()` takes the ordinal/codepoint, then passing in a component from the encoded form reflects a misunderstanding, not a technical limitation. Is the misunderstanding mine? Is `CHAR()` supposed to accept an _encoded byte_ value? If so, then it's not just a disparity relative to `NCHAR()` but a semantic deviation.